reasoner/controller/NestedConjunctionController.java

```diff
@@ -34,18 +36,11 @@ public class NestedConjunctionController extends ConjunctionController<
         NestedConjunctionController.NestedConjunctionProcessor
         > {
 
-    public NestedConjunctionController(Driver<NestedConjunctionController> driver, Conjunction conjunction,
+    public NestedConjunctionController(Driver<NestedConjunctionController> driver, ResolvableConjunction conjunction,
                                        Context context) {
         super(driver, conjunction, context);
     }
 
-    @Override
```

Comment:
No longer needed, since these can be answered by the conjunction in all subclasses of ConjunctionController.

Reply:
Outdated; This has been retained